src/components/Header/index.jsx

```diff
@@ -137,6 +140,16 @@ const GordonHeader = ({ onDrawerToggle }) => {
     </Button>
   );
 
+  function logoSize() {
```

Comment:
"logoSize()" sounds like it returns the size of the logo, instead of returning the right sized logo.  How about renaming it something like logoSizedForHeader() or headerLogo().